specification/under_development/profiles/Base_Profile.md

```diff
@@ -0,0 +1,522 @@
+measurements at the end of the program. More details about each of the bullets
+are outlined below.
+
+**Bullet 1: Quantum transformations** <br/>
+
+The set of available instructions that transform the quantum state may vary
+depending on the targeted backend. The profile specification defines how to
+leverage and combine the available instructions to express a program, but does
+not dictate which quantum instructions may be used. Targeting a program to a
+specific backend requires choosing a suitable profile and quantum instruction
+set (QIS). Both can be chosen largely independently, though certain instruction
+sets may be incompatible with this (or other) profile(s). The section on the
+[quantum instruction set](#quantum-instruction-set) defines the requirements for
+a QIS to be compatible with the Base Profile. More information about the role of
+the QIS, recommendations for front- and backend providers, as well as the
+distinction between runtime functions and quantum instructions can be found in
+[this document](../Instruction_Set.md).
+
+**Bullet 2: Measurements** <br/>
+
+The second requirement should be taken to mean that a Base Profile compliant
+program does *not* apply instructions to a qubit after it has been measured;
+instructions that result in a unitary transformation of the quantum state must
+be applied before performing any irreversible actions such as measurements. It
+specifically also implies the following:
+
+- There is no need for the quantum processor ([QPU](../Execution.md)) to be able
+to measure only a subset of all available qubits at a time.
+- Executing a Base Profile compliant program does not require support for
+applying quantum instructions dependent on measurement outcomes.
+
+**Bullet 3: Program output** <br />
+
+The specification of QIR and all its profiles needs to permit to accurately
+reflect the program intent. This includes being able to define and customize the
+program output. The Base Profile specification hence requires explicitly
+expressing which values/measurements are returned by the program and in which
+order. How to express this is defined in the section on [output
+recording](#output-recording).
+
+While it is sufficient for the QPU to do a final measurement of all qubits in a
+predefined order at the end of the program, only the selected subset will be
+reflected in the produced output schema. A suitable output schema can be
+generated in a post-processing step after the computation on the quantum
+processor itself has completed; customization of the program output hence does
+not require support on the QPU itself.
+
+The defined [output schemas](../output_schemas/) provide different options for
+how a backend may express the computed value(s). The exact schema can be freely
+chosen by the backend and is identified by a string label in the produced
+schema. Each output schema contains sufficient information to allow quantum
```

Comment:
Perhaps I'm having a disconnect here, but Line 416 says that the `"output_labeling_schema"` attribute value can be freely chosen by the compiler _frontend_, but this comment makes it sound like the _backend_ is deciding on the schema. In either case, is it freely chosen from a set of limit output options (like `Labeled_And_Async` and `No_Labels_And_Ordered`)? If so, I think those options should probably enumerated in a table. If not, then it seems like it is an arbitrary string that the front-end can choose, but then the backend may not know what to do with that attribute value.